SoftLayer/managers/image.py

```diff
@@ -120,28 +120,68 @@ def edit(self, image_id, name=None, note=None, tag=None):
 
         return bool(name or note or tag)
 
-    def import_image_from_uri(self, name, uri, os_code=None, note=None):
+    def import_image_from_uri(self, name, uri, os_code=None, note=None,
+                              ibm_api_key=None, root_key_id=None,
+                              wrapped_dek=None, kp_id=None, cloud_init=None,
+                              byol=None, is_encrypted=None):
```

Comment:
Default values for these cloud_init, byol, and is_encrypted params should be Boolean False values, which would then match the description for them below.

Reply:
This will be included in the next patch.